sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -56,6 +64,11 @@ class CustomerIO internal constructor(
             return this
         }
 
+        fun shouldAutoRecordScreenViews(shouldRecordScreenViews: Boolean): Builder {
```

Comment:
If you want to keep things consistent, in the iOS SDK we [`autoTrackScreenViews` as the function name](https://github.com/customerio/customerio-ios/blob/4db3dc097e3cf127264ef2d0830499bd10966b4d/Sources/Tracking/Store/SdkConfig.swift#L25) for configuration.